### ISSUE TITLE
[bug]: Self-host setup, EMAIL_FROM from env file is not used

### ISSUE THREAD

**Issue by olivierlambert** [makeplane/plane#3123]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

Editing the `.env` file to put:
```
EMAIL_FROM="My team <admin@mydomain.org>"
```

Then restarting/stopping & starting or even full reboot: my message is always rejected because the "from" is always `team@mailer.plane.so` regardless the content of `EMAIL_FROM` in my `.env` file.

However, the rest is correctly taken into account, because it goes to the right SMTP server via user/password, TLS and such. Only the `EMAIL_FROM` is ignored.

### Steps to reproduce

1. Change the .env file to use a "custom" `EMAIL_FROM`
2. Restart the containers
3. Send an email
4. Always using the default `EMAIL_FROM`

### Browser

Mozilla Firefox

### Version

Self-hosted

**Comment by olivierlambert** [comment]
I think I might have a lead:

1. If I clean the email config in God mode, emails aren't sent anymore
2. So it's probably that the `.env` file isn't used to send emails, so it would be then needed to update the `/god-mode/email` UI to add a "from" field so it's took into account.

**Comment by Heucles** [comment]
You are correct, if you go into the file inside the container:

```
 docker exec -it plane-app-worker-1 /bin/bash
```
```

 vi plane/bgtasks/workspace_invitation_task.py 
``` 

and hard code your EMAIL_FROM like this

```python
        msg = EmailMultiAlternatives(                                                                                              
            subject=subject,                                                                                                       
            body=text_content,                                                                                                     
            from_email="samplemail@xxxxx.ai",                                                                 
            to=[email],                                                                                                            
            connection=connection,                                                                                                 
        )                                                                                                                          
        msg.attach_alternative(html_content, "text/html")                                                                          
        msg.send()                                                           
                      
```

I works!!!

**Comment by pablohashescobar** [comment]
Thanks for flagging this we will push a patch soon.

**Comment by olivierlambert** [comment]
\o/ Thanks for the hint, and I'm happy to see I spotted an actual issue, helping you guys to make a better product. Congrats again for this excellent project :+1:

**Comment by olivierlambert** [comment]
I followed @Heucles instructions, and I confirm it worked (as a quick'n'dirty workaround)

**Comment by vihar** [comment]
@olivierlambert, this issue has been fixed on our latest release, `v0.14.1-dev`. If you're seeing any issues, please let us know.

**Comment by olivierlambert** [comment]
I can confirm it's fixed! Thank you very much.
